Sources/XcbeautifyLib/Pattern.swift

```diff
@@ -423,6 +423,10 @@ enum Pattern: String {
     /// $1 = whole error
     case xcodebuildError = #"^(xcodebuild: error:.*)$"#
 
+    /// Regular expression captured groups:
+    /// $1 = whole warning, excluding the path to the .xcodeproj file
+    case xcodeprojWarning = #".*.xcodeproj: (warning:.*)$"#
```

Comment:
Not that I'm looking at the raw output again, I think this can be merged with the `compileWarning`, and `xcodeprojError` can be merged with `compileError`, since it's the same `<file path>: (warning|error): <message>`  pattern. What do you think?

Reply:
Possibly, yes, but the strings I was testing with were not being caught by `compileError` and `compileWarning`. Let me see if I can figure out why.

Reply:
Awesome! This is a much better solution than adding more patterns. Thank you.